delphi/translators/for2py/rectify.py

```diff
@@ -521,73 +526,74 @@ def handle_tag_body(
 
                         if parent.tag == "loop":
                             if (
-                                child.tag == "if"
-                                or child.tag == "statement"
-                                and "conditional-goto-stmt" in child.attrib
+                                    child.tag == "if"
+                                    or child.tag == "statement"
+                                    and "conditional-goto-stmt" in child.attrib
                             ):
                                 self.goto_under_loop = True
                     else:
                         new_parent = current
                         # Reconstruction of statements
                         if (
-                            "parent" in current.attrib
-                            and (not self.goto_under_loop
-                            and current.attrib['parent'] == "program")
-                            or (self.goto_under_loop
-                            and current.attrib['parent'] == "loop")
+                                "parent" in current.attrib
```

Comment:
parenthesize to make the operator grouping easy to understand.

Reply:
Relevant parenthesis has been added but since this part of this code seems to be for go-tos and written by @hlim1 , I would like him to confirm this change.

Reply:
I confirm that the grouping was done appropriately.